Bottom: cda603223ee1c88851213ed96ad9af6bb1b95ea0
Top:    ff75be80116612759537456aa15559641ad61c86
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 16:27:14 -0500

Update openldap scripts to run behind reverse proxy


---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index c605ebaa2..de98b2d99 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -131,15 +131,14 @@
     detach: true
     env:
       PHPLDAPADMIN_HTTPS: 'false'
-      PHPLDAPADMIN_LDAP_HOSTS: 'ldap'
+      PHPLDAPADMIN_LDAP_HOSTS: '{{ openldap_container_name }}'
+      PHPLDAPADMIN_TRUST_PROXY_SSL: 'true'
       LDAP_HOSTNAME: '{{ openldap_hostname }}'
-    hostname: '{{ phpldapadmin_hostname }}'
+    hostname: '{{ phpldapadmin_container_name }}'
     image: '{{ phpldapadmin_image_name }}:{{ docker_image_tag }}'
     name: '{{ phpldapadmin_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
-        links:
-          - '{{ openldap_container_name }}:ldap'
     ports: '{{ phpldapadmin_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
